src/app/Library/CrudPanel/Traits/Query.php

```diff
@@ -142,4 +142,13 @@ public function count()
     {
         return $this->query->count();
     }
+
+    public function applyCrudOrder($column_name, $column_direction = 'ASC')
```

Comment:
Maybe a better name for this would be `rawOrderBy()`? Because currently it's like calling `CrudPanel::applyCrudOrder()` or `$crud->applyCrudOrder()` which seems a bit redundant.

Reply:
Oups, sorry. We already have a `orderByRaw()` so that could get confusing. Ok then, how about `orderByWithPrefix()`? That should be nice and different and explicit, right?

Reply:
I vote for `orderByWithPrefix` 😅